s3/locals.tf

```diff
@@ -4,4 +4,12 @@ locals {
   standard_bucket_name = "${var.project}-${var.function}-${local.environment}"
   global_bucket_name   = "${var.project}-${var.function}"
   bucket_name          = var.environment_suffix == true ? local.standard_bucket_name : local.global_bucket_name
+  sns_topic_arn        = var.sns_topic_arn == "" ? "arn:aws:sns:${var.sns_topic_region}:${data.aws_caller_identity.current.account_id}:${var.project}-logs-${local.environment}" : var.sns_topic_arn
```

Comment:
Should this be named "log_aggregation_sns_topic_arn" as likely will need to add another sns topic for the backend checks

Reply:
agreed, now have separate local entries for the log data SNS topic, and the main S3 bucket SNS topic